app/handlers/base.py

```diff
@@ -142,6 +142,7 @@ def Session(self):
             # must merge the user object with the current session
             # ref: https://docs.sqlalchemy.org/en/14/orm/session_basics.html#adding-new-or-existing-items
             session.add(self.current_user)
+            session.bind = DBSession.session_factory.kw["bind"]
```

Comment:
Looks good, thanks Michael. What is this line for?

Reply:
@stefanv I find that the explicit bind is necessary in this new 2.0 setup!

Reply:
what is actually the value of this `kw["bind"]`?